editor/editor_settings.cpp

```diff
@@ -573,6 +573,7 @@ void EditorSettings::_load_defaults(Ref<ConfigFile> p_extra_config) {
 	_initial_set("editors/2d/bone_outline_color", Color(0.35, 0.35, 0.35));
 	_initial_set("editors/2d/bone_outline_size", 2);
 	_initial_set("editors/2d/viewport_border_color", Color(0.4, 0.4, 1.0, 0.4));
+	_initial_set("editors/2d/remove_viewport_limits", false);
```

Comment:
I would go for something opposite. Like "Constraint the editor view to its content." defaulting to `true`.

Reply:
Is `limit_editor_view` acceptable?

Reply:
I'd use "constraint", it's more positive. ^^ The thing is that it is ok as an identifier, but the text in the settings isn't really explicit. I would be good to be able to customize it (but that's out of the scope of this PR, so constraint_editor_view should be enough for now).

Reply:
Note that the verb is "constrain" :)